agv2:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50  

  # Position Controllers ---------------------------------------
  rod_front_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rod_front_joint
    pid: {p: 1000.0, i: 0.01, d: 10.0}
  rod_rear_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: rod_rear_joint
    pid: {p: 1000.0, i: 0.01, d: 10.0}
